documentation/source/index.rst

```diff
@@ -24,18 +24,23 @@ Terminology
 
 package
   A blob of files that can be unpacked into a directory and which has a
-  ``dylan-package.json`` file in the top-level directory which describes its
-  attributes. A package currently corresponds to a single Git repository. A
-  package has a set of versioned releases.
+  ``dylan-package.json`` file in the top-level directory. A package currently
```

Comment:
Suggest using `:file:` role

Reply:
Neat. Let me defer this for now, since I need to update all file references. I had been trying to decide between ``foo``, "foo", and foo for files and this answers that question.